cfgov/login/migrations/0004_simplify_password_history.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.2.18 on 2023-04-12 17:37
```

Comment:
Minor maybe nitpicky thing - we could collapse this migration with the 0003 one you made into a single "simplify_password_logic" one.